spec/hello_sign/client_spec.rb

```diff
@@ -28,7 +28,8 @@
           :logging => false,
           :proxy_uri => 'proxy_uri',
           :proxy_user => 'proxy_user',
-          :proxy_pass => 'proxy_pass'
+          :proxy_pass => 'proxy_pass',
+          :timeout => 240
```

Comment:
@jyoung488 this is the place that tests that the value gets passed through from the config and the client, which is good. I think the only other thing I would test is that the client params get passed on to Faraday, I don't see that currently for any of them, but I think it would be worthwhile. I don't think we have to test that the timeout works, we can trust Faraday!

Reply:
Thank you! Added a couple tests for the timeout parameter and merged 👍